app/src/main/java/com/doctor/yumyum/presentation/base/activity/BaseActivity.kt

```diff
@@ -0,0 +1,30 @@
+package com.doctor.yumyum.presentation.base.activity
+
+import android.os.Bundle
+import android.widget.Toast
+import androidx.annotation.LayoutRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+
+abstract class BaseActivity<T : ViewDataBinding>(@LayoutRes private val layoutResId: Int) :
+    AppCompatActivity() {
+    protected lateinit var binding: T
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = DataBindingUtil.setContentView(this, layoutResId)
+    }
+
+    protected fun showCustomToast(message: String) {
+        Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
+    }
```

Comment:
```suggestion
    protected fun showToast(message: String) {
        Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
    }
```
`showToast` 라는 네이밍으로도 충분히 직관적일 듯 해 제안드립니다.